.github/workflows/format-code.yml

```diff
@@ -29,13 +29,12 @@ jobs:
           git clone $HEAD_REPO .
           git checkout -b "$HEAD_REF" "origin/$HEAD_REF"
 
-      - name: "Install indent"
+      - name: "Update"
         run: |
           sudo apt-get -qq update
-          sudo apt-get install -y indent
```

Comment:
Does `clang-format` need to be installed or is it already in the base image? If it's already in the base image, we can remove the `sudo apt-get --qq update` since it just updates the package database.

Reply:
Looks like version 11 is in the base image on the runner 🙂